scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -54,9 +56,13 @@ workflow GvsCreateFilterSet {
   String fq_sample_table = "~{project_id}.~{dataset_name}.sample_info"
   String fq_alt_allele_table = "~{project_id}.~{dataset_name}.alt_allele"
   String fq_info_destination_table = "~{project_id}.~{dataset_name}.filter_set_info"
+  String fq_info_destination_table_vqsr_lite = "~{project_id}.~{dataset_name}.vqsr_lite_filter_set_info"
```

Comment:
nit: Could we name the table 'filter_set_info_vqsr_lite' (or something like that) so that when I browse with the bigQuery console, that table is listed side-by-side with the other filter table?

Reply:
I'm fine with this myself, as I'm not particularly wed to the name.  It's just vqsr_lite_filter_set_info is just the name that was put into the ticket, so it was the one used.  Any strong opinion on the matter, @rsasch?

Reply:
I'm going to consider this resolved, as George appears to have changed the naming in his follow up PR.  And until his PR is merged, the naming does not matter.